### PR TITLE
Release 4.7.0

### PR DESCRIPTION
**Pull request by johnhwhite**
- Updated filter and sort buttons use modern theme icons. [#222](https://github.com/blackbaud/skyux-lists/pull/222)

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-lists?_sp.spa=pr-release-4.7.0](https://developer.blackbaud.com/skyux-lists?_sp.spa=pr-release-4.7.0)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 4.7.0 (2021-04-01)
+
+- Updated filter and sort buttons use modern theme icons. [#222](https://github.com/blackbaud/skyux-lists/pull/222)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md:3]:
```suggestion
- Updated the filter and sort buttons to use modern theme icons. [#222](https://github.com/blackbaud/skyux-lists/pull/222)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# 4.7.0 (2021-04-01)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md:1]:
```suggestion
# 4.7.0 (2021-04-02)
```
Assuming we won't actually merge this until tomorrow. If you catch it tonight, disregard this edit.

**Review by blackbaud-johnly — APPROVED** [review]

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-lists?_sp.spa=pr-release-4.7.0](https://developer.blackbaud.com/skyux-lists?_sp.spa=pr-release-4.7.0)

**Pull request merged**